generators/client/templates/angular/src/main/webapp/app/admin/_index.ts

```diff
@@ -41,3 +41,4 @@ export * from './user-management/user.model';
 export * from './user-management/user-modal.service';
 <%_ } _%>
 export * from './admin.state';
+export * from './admin.module';
```

Comment:
@gmarziou was this really required? I removed module from exports as it was causing circular dependencies as most of the above components will be imported into the module as well.